src/lava/utils/profiler.py

```diff
@@ -22,7 +22,8 @@
 from lava.magma.runtime.runtime import Runtime
 from lava.magma.core.process.message_interface_enum import ActorType
 from lava.magma.compiler.compiler import Compiler
-from lava.magma.core.resources import Loihi1NeuroCore, Loihi2NeuroCore
+from lava.magma.core.resources import (
+    AbstractComputeResource, Loihi1NeuroCore, Loihi2NeuroCore)
 
 
 class LavaProfiler(ABC):
```

Comment:
Why LavaProfiler? Profiler should be enough.

Why does this have to be an ABC? In this case I would probably call it AbstractProfiler.